.changeset/fifty-clouds-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Supports symlinked content collection directories
```

Comment:
```suggestion
Fixes a case where symlinked content collection directories were not correctly resolved
```

A suggestion to hint that this is a bug fix. The previous phrase seemed more like a new feature